target/linux/mvebu/config-4.4

```diff
@@ -403,3 +403,11 @@ CONFIG_ZBOOT_ROM_TEXT=0x0
 CONFIG_ZLIB_DEFLATE=y
 CONFIG_ZLIB_INFLATE=y
 CONFIG_ZONE_DMA_FLAG=0
+CONFIG_MMC_BLOCK=y
+CONFIG_MMC_SDHCI=y
+CONFIG_MMC_SDHCI_PLTFM=y
+CONFIG_MMC_SDHCI_PXAV3=y
+CONFIG_MMC_SDHCI_F_SDH30=n
+CONFIG_MMC_SDHCI_OF_ARASAN=n
+CONFIG_MMC_SDHCI_OF_AT91=n
+CONFIG_MMC_SDHCI_PCI=n
```

Comment:
I'm talking about the openwrt make target. You have the target `kernel_oldconfig`, `kernel_menuconfig` and `kernel_defconfig` available that work similar to the `oldconfig`, `menuconfig` and `defconfig` targets, except that they work on the kernel config, and update the config in `target/linux/<target>/config-<version>`.